### DISCUSSION TITLE
(Done) Dec 11th 2024: Celebrating A Year of (Madison) Clojure

### DISCUSSION THREAD

**Discussion by frenchy64** [madclj/madclj.com#7] [Announcements]
👍 to RSVP

https://madclj.com/

Wednesday, December 11, 2024
6:30 PM to 9:00 PM CT

[StartingBlock Madison](https://www.google.com/maps/search/?api=1&query=43.081245%2C%20-89.374054)
821 E Washington Ave 2nd floor · Madison, WI

It's been a busy year! This will be our 11th and final meetup of 2024.

Let's wind down and celebrate the year together. We'll have a loosely structured discussion about our year with an eye towards Madison Clojure in 2025.

Topics:
- what characterized Clojure in 2024 for you?
- feedback on Madison Clojure in 2024
- planning 2025

Whether this is streamed will be determined closer to the event.

Agenda:
- 6:30pm arrive, food & drink provided (TBD)
- 7pm start structured discussion
- 8pm end structured discussion
- 9pm end meetup

**Comment by djwhitt** [comment]
Sorry, had to remove my :+1:. I won't make it. I've been fighting some ops fires at work and am pretty exhausted.

**Comment by frenchy64** [comment]
Pizza will be there around 7pm, I'll try and find some snacks to hold us over. I should be able to let people in at 6:30pm.

**Comment by dpetranek** [comment]
Here's the Tree Calculus links:

https://treecalcul.us/
https://github.com/barry-jay-personal/tree-calculus/blob/master/tree_book.pdf

**Comment by frenchy64** [comment]
![signal-2024-12-11-222546_002](https://github.com/user-attachments/assets/60014a76-fddb-4933-b67a-c20e10b72e23)

**Comment by frenchy64** [comment]
https://github.com/tonsky/fast-edn
https://github.com/djblue/portal
